master/internal/core_experiment.go

```diff
@@ -293,7 +293,13 @@ func (m *Master) parseCreateExperiment(ctx context.Context, req *apiv1.CreateExp
 	}
 	workspaceID := resolveWorkspaceID(workspaceModel)
 	isSingleNode := resources.IsSingleNode() != nil && *resources.IsSingleNode()
-	poolName, _, err := m.ResolveResources(resources.ResourcePool(), resources.SlotsPerTrial(), workspaceID, isSingleNode)
+	isUnmanaged := req.Unmanaged != nil && *req.Unmanaged
```

Comment:
there is `GetUnmanaged` on `req` that is autogenerated by protos you could use. it's a bit subtle though that it only works because the zero value for a bool is the correct default.